projects/demo/src/modules/info/testing/testing.template.html

```diff
@@ -0,0 +1,137 @@
+                </li>
+                <li class="tui-list__item">
+                    To disable animation in ALL Taiga UI component we use token
+                    <code>TUI_ANIMATIONS_DURATION</code>.
+                </li>
+            </ol>
+
+            <tui-doc-code
+                filename="app.module.ts"
+                [code]="disableAnimationExample"
+            ></tui-doc-code>
+        </li>
+
+        <li id="screenshot-bot">
+            <strong
+                >Use our
+                <a
+                    tuiLink
+                    target="_blank"
+                    href="https://github.com/apps/argus-bot"
+                    >Github App Argus</a
+                ></strong
+            >
+
+            <p>
+                Using official Cypress dashboard is rather expensive (especially
+                if you run thousands of tests every day). What's why we
+                developed Github App. This bot:
+            </p>
+
+            <ul class="tui-list">
+                <li class="tui-list__item">
+                    Downloads artifacts from workflow with tests, finds
+                    screenshots diffs images, and pins them to the tests failure
+                    report as Pull Request comment.
+                </li>
+                <li class="tui-list__item">
+                    Always holds first PR comment. Any workflow updates (for
+                    example, new commit pushed to the branch) edit already
+                    existing bot comment. No endless stream of comments from
+                    bot!
+                </li>
+                <li class="tui-list__item">
+                    Doesn't require any data-storage for your screenshots with
+                    diffs. Bot creates separate branch to upload images for its
+                    tests failure report. After closing PR bot removes all saved
+                    screenshots. Also, you can delete this branch at any time to
+                    clean github history.
+                </li>
+            </ul>
+
+            <ol class="steps">
+                To begin using
+                <strong>Argus bot</strong
+                >:
+
+                <li>
+                    <a
+                        tuiLink
+                        target="_blank"
+                        href="https://github.com/apps/argus-bot/installations/new"
+                        >Invite him</a
+                    >
+                    to you repo.
+                </li>
+                <li>
+                    See its configurable params
+                    <a
+                        tuiLink
+                        target="_blank"
+                        href="https://github.com/TinkoffCreditSystems/argus"
+                        >in its repo's README</a
+                    >
+                    or use default ones.
+                </li>
+            </ol>
+        </li>
+    </ol>
+</tui-doc-page>
```

Comment:
Maybe make separate subpages for token and argument?
And I think we can add argus to friendly libraries page

Reply:
done